packages/waku/src/cli.ts

```diff
@@ -46,6 +46,9 @@ const { values, positionals } = parseArgs({
     'with-aws-lambda': {
       type: 'boolean',
     },
+    partial: {
```

Comment:
I feel like I'd like to start with `--experimental-partial` first, and get some user feedback.